docs/release.md

```diff
@@ -1,43 +1,65 @@
 # Release Guidelines
 
-## Introduction
+Welcome to Eclipse Tractus-X Release Guidelines, known as TRGs. With Release Guidelines, we are defining quality
+criteria for our repositories, products and artifacts in general. TRGs are numbered and grouped into sections, to
+provide an easy-to-use overview. The git history of the TRGs can be use to browse changes made throughout the project
+history. You can also conduct the [TRG Changelog](release/trg-0) to see when new TRGs have been added.
```

Comment:
I would reframe it a bit:

```suggestion
history. You can also check the [TRG Changelog](release/trg-0) to get better view on history of new/changed/deprecated TRGs.
```

Reply:
[7632dba](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/871/commits/7632dba036160282c9455be557dc103ef7019e79)